src/particle/particle/particle.py

```diff
@@ -1289,6 +1297,10 @@ def from_string_list(cls, name):
             return []
 
     @classmethod
+    @deprecated(
+        version="0.16.0",
+        reason="This method is deprecated and will be removed from version 0.17.0. Use finditer or findall instead.",
+    )
     def _from_group_dict_list(cls, mat):
```

Comment:
No one should use this anyway, right? It's private, so not sure if a deprecation is needed? Maybe it's confusing if you see it as a user

Reply:
You're totally write. I was "trigger happy" it seems. Rolling this back.